lib/broadside/target.rb

```diff
@@ -93,5 +93,16 @@ def to_h
         Cluster: @cluster
       }
     end
+
+    def update_safe_service_config
+      attributes_disallowed_for_updates = %i(
```

Comment:
constant?  `SERVICE_ATTRIBUTES_ONLY_SET_AT_INSTANTIATION`?

(i don't love that name, feel free to rename, but i do think it should be a constant)